hfile/go_reader.go

```diff
@@ -0,0 +1,10 @@
+// Copyright (C) 2017 Foursquare Labs Inc.
+// +build native_snappy
```

Comment:
maybe add a quick comment as to why this happens? just in case we somehow stop working on this and forget forever